integrations/wiz/.port/resources/port-app-config.yaml

```diff
@@ -16,7 +16,9 @@ resources:
             description: .description
   - kind: issue
     selector:
-      query: 'true'
+      query: "true"
+      maxPages: 20
+      statusList: ["OPEN", "IN_PROGRESS"]
```

Comment:
Can we rename it to statuses?

Reply:
Let's let it be. statuses is less easy to enunciate when spoken. I prefer this.